src/app.py

```diff
@@ -19,10 +19,8 @@
 # Log rotation is handled via logrotate on the host system with a configuration file
 # Do NOT handle log file and rotation via the Python logging to avoid issues with multi-worker processes
 
-logging.basicConfig(format='[%(asctime)s] %(levelname)s in %(module)s:%(lineno)d: %(message)s', level=logging.DEBUG,
```

Comment:
Need to revert the change, DEBUG level is being used and it's helpful for troubleshooting.

Reply:
Logging level reverted to DEBUG in branch file.